app/models/concerns/host_info_extensions.rb

```diff
@@ -15,8 +15,6 @@ module HostInfoExtensions
   included do
     # Add default providers
     HostInfo.register_info_provider(HostInfoProviders::StaticInfo)
-    HostInfo.register_info_provider(HostInfoProviders::ConfigGroupsInfo)
-    HostInfo.register_info_provider(HostInfoProviders::PuppetInfo)
```

Comment:
I really dislike that this removes the `puppetmaster` and `puppet_ca` parameters. Especially since those remain in the host model. I also think you'll drop the ip, ip6 and mac.